.github/workflows/package-linux.yml

```diff
@@ -1,6 +1,9 @@
 name: package-linux
 
 on:
+  pull_request:
+    paths:
+      - .github/workflows/package-linux.yml
```

Comment:
This will be removed before merge, right ?

Reply:
no

Reply:
I want to keep this trigger, since it will be only triggered when we create a PR that edit the workflow's file